README.md

```diff
@@ -91,6 +91,31 @@ class _MyWidgetState extends State<MyWidget> {
 }
 ```
 
+#### Native Auth example
+```dart
```

Comment:
Could we add a warning here that says the current native auth implementation is experimental and there might be API changes in the future?